# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: taskcluster/ci/wgpu/kind.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/taskcluster/ci/wgpu/kind.yml b/taskcluster/ci/wgpu/kind.yml
deleted file mode 100644
--- a/taskcluster/ci/wgpu/kind.yml
+++ /dev/null
@@ -1,53 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
----
-loader: taskgraph.loader.transform:loader
-
-kind-dependencies:
-    - toolchain
-    - build
-
-transforms:
-    - taskgraph.transforms.job:transforms
-    - taskgraph.transforms.task:transforms
-
-job-defaults:
-    attributes:
-        retrigger: true
-    run-on-projects: ['mozilla-beta', 'trunk']
-    treeherder:
-        tier: 1
-        kind: other
-    worker:
-        max-run-time: 1800
-        env:
-            RUST_BACKTRACE: 'full'
-            RUSTFLAGS: '--deny warnings'
-    run:
-        sparse-profile: wgpu
-    when:
-        files-changed:
-            - 'gfx/wgpu/**'
-
-jobs:
-    linux64-debug:
-        description: Runs debug-mode wgpu CI tests on a Linux worker
-        worker-type: b-linux
-        worker:
-            docker-image: {in-tree: wgpu}
-        fetches:
-            toolchain:
-                - linux64-rust
-                - wgpu-deps
-        run:
-            using: run-task
-            command: >-
-                export PATH=$PATH:$MOZ_FETCHES_DIR/rustc/bin &&
-                cd $GECKO_PATH/gfx/wgpu &&
-                mv $MOZ_FETCHES_DIR/wgpu-deps/{vendor,.cargo} ./ &&
-                cargo test --verbose --frozen
-        treeherder:
-            platform: linux64-qr/debug
-            symbol: Wgpu(test)
-            kind: other